### ISSUE TITLE
Icons unnecessarily read by screen reader

### ISSUE THREAD

**Issue by kangabell** [danny-avila/LibreChat#5724]
Various icons throughout the app are being sometimes read by a screen reader when they should be hidden.

Examples from Prompts Panel:
- The plus sign in the "+ Create Prompt" button is being read by the screen reader as "plus".
- The "(x)" next to the Variables heading reads as just "image"
- The left arrow before the "Back to Chat" link reads as "image"

## Testing Method
Manual – VoiceOver, Chrome, MacOS

## Recommended Action
Add an empty alt attribute `(alt="")` or `aria-hidden="true"` to allow the screen reader to ignore any icons that don't need to be read.
